src/main/java/org/apache/mesos/scheduler/plan/api/CurrentlyActiveInfo.java

```diff
@@ -60,7 +63,7 @@ public CurrentlyActivePhaseInfo getPhaseStatus() {
       return phaseStatus;
     }
 
-    @JsonProperty("stage")
+    @JsonProperty("plan")
```

Comment:
I looked around in Kafka and Cassandra's docs and see no mention.  The endpoint was always `v1/plan` and the word `stage` never showed up in that output.